Ical.Net/Evaluation/RecurrencePatternEvaluator.cs

```diff
@@ -378,14 +392,14 @@ private List<int> GetByWeekNoForYearNormalized(RecurrencePattern pattern, int ye
     /// </summary>
     /// <param name="dates">The list of dates to which the BYYEARDAY rules will be applied.</param>
     /// <returns>The modified list of dates after applying the BYYEARDAY rules.</returns>
-    private List<DateTime> GetYearDayVariants(List<DateTime> dates, RecurrencePattern pattern, bool? expand)
+    private List<DateTime> GetYearDayVariants(List<DateTime> dates, RecurrencePattern pattern, bool? expand, ExpandContext expandContext)
```

Comment:
Make method `static`?

Reply:
I know CodeCov is complaining. I left it non-static intentionally, to stay consistent with the other related methods. Nothing new also.